src/Common/StringSearcher.h

```diff
@@ -569,8 +569,10 @@ class StringSearcher<true, ASCII> : public StringSearcherBase
     uint8_t first{};
 
 #ifdef __SSE4_1__
-    /// vector filled `first` for determining leftmost position of the first symbol
-    __m128i pattern;
+    uint8_t second{};
+    /// vector filled `first` or `second` for determining leftmost position of the first and second symbols
```

Comment:
Variable names `first_pattern` and `second_pattern` are too generic. Let's change them to `first_needle_character_vec` and `second_needle_character_vec`. Let's also change the comment (l. 573) to `The first/second needle character broadcasted into a 16 bytes vector`.

Reply:
Done.